client/src/UserFunctions.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+export const registerFunc = newUser => {
+    return axios.post('http://localhost:5000/register', {
```

Comment:
1) move
http://localhost:5000 to env variables

Reply:
2. Create axious instanse to http://localhost:5000 in separate file and just import it here in order to call path /register